content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/manually.rst

```diff
@@ -0,0 +1,109 @@
+and the required quantity **must** be reserved manually.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
+Configuration
+=============
+
+To set the reservation method to *Manually*, navigate to :menuselection:`Inventory app -->
+Configuration --> Operations Types`.
+
+In the :guilabel:`General` tab, locate the :guilabel:`Reservation Method` field, and select
+:guilabel:`Manually`.
+
+.. image:: manually/manually-operations-type.png
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+Workflow
+========
+
+To see the *Manually* reservation method in action, create a new :abbr:`SO (sales order)` by
+navigating to :menuselection:`Sales app --> New`.
+
+Add a customer in the :guilabel:`Customer` field. Then, in the :guilabel:`Order Lines` tab, click
+:guilabel:`Add a product`, and select a product to add to the quotation from the drop-down menu.
+Finally, in the :guilabel:`Quantity` column, adjust the desired quantity of the product to sell.
+
+Once ready, click :guilabel:`Confirm` to confirm the sales order.
+
+Click the green :guilabel:`📈 (forecast graph)` icon on the product line to reveal the product's
+:guilabel:`Availability` tooltip. This tooltip reveals the reserved number of units for this order.
+Because the reservation method is set to *Manually*, the :guilabel:`Reserved` quantity reads `0
+Units`.
+
+However, below that quantity reads `Available in stock`. This is because the quantity is available,
+but must be manually reserved.
+
+.. image:: manually/manually-availability-tooltip.png
+   :align: center
+   :alt: Confirmed sales order with product availability tooltip selected.
+
+Once the :abbr:`SO (sales order)` is confirmed, navigate to the :menuselection:`Inventory app`, and
+locate the :guilabel:`Delivery Orders` card on the :guilabel:`Inventory Overview` page.
+
+The :guilabel:`Delivery Orders` card displays the current status of live orders, including those
+with a *Waiting* status. Orders with this status indicate that the products in those orders have
+either not been reserved yet, or are not in stock at all.
+
+.. image:: manually/manually-delivery-orders-card.png
+   :align: center
+   :alt: Delivery orders task card with waiting status orders.
+
+To see the :abbr:`SO (sales order)` created previously, click the :guilabel:`X Waiting` button on
+the card (in this case, `8 Waiting`).
+
+Locate the delivery order (DO) tied to the :abbr:`SO (sales order)` that was previously created, and
+click the line to view it.
+
+On the :guilabel:`Delivery Order` form, the status in the :guilabel:`Product Availability` field is
+listed as `Available`, in yellow text instead of green. This is because there is sufficient stock on
+hand for this order, but no quantity has been reserved yet.
+
+In the :guilabel:`Operations` tab on the :guilabel:`Product` line, the numbers in the
```

Comment:
```suggestion
In the :guilabel:`Operations` tab, on the :guilabel:`Product` line, the numbers in the
```
added comma for enhanced sentence clarity

Reply:
Suggestion implemented ✅